src/modm/platform/gpio/rpi/module.lb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Erik Henriksson
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+
+def init(module):
+    module.name = ":platform:gpio"
+    module.description = "Hosted GPIO for Raspberry Pi"
+
+
+def prepare(module, options):
+    if not options[":target"].has_driver("gpio"):
```

Comment:
```suggestion
    if not options[":target"].has_driver("gpio:rpi"):
```

Reply:
Nope 😉

Reply:
Meh, forgot to save the file in my editor :(